src/main/java/br/ufpe/cin/mergers/SemistructuredMerge.java

```diff
@@ -32,6 +33,15 @@
 	static final String MERGE_SEPARATOR = "##FSTMerge##";
 	static final String SEMANTIC_MERGE_MARKER = "~~FSTMerge~~";
 
+	private static final List<ConflictHandler> CONFLICT_HANDLERS = ImmutableList.of(
+			new TypeAmbiguityErrorHandler(),
+			new NewElementReferencingEditedOneHandler(),
+			new RenamingConflictsHandler(),
+			new InitializationBlocksHandler(),
+			new DuplicatedDeclarationHandler(),
+			new DeletionsHandler()
+	);
+
```

Comment:
so the idea is to load all handlers independently of command line options? I thought that we would load handlers by demand, accordingly to command line options

Reply:
I think parameters are not taken in consideration here yet because the related pull request was not merged, right @gaabs  and @jvcoutinho  ?

Reply:
That's right, @guilhermejccavalcanti . We should still decide the best approach.

One could filter this list using the command line options provided, for instance.